### PR TITLE
Move to aiomysql pool/connection broker

### PR DESCRIPTION
**Pull request by Jetsurf**
Massive change to mysql handling. Make use of aiomysql and connection pools.

### CODE REVIEW

File: modules/nsotoken.py [hunk 1]
```diff
@@ -1,76 +1,74 @@
...
-	def get_iksm_token_mysql(self, userid):
+	async def get_iksm_token_mysql(self, userid):
+		cur = await self.sqlBroker.connect()
 		stmt = "SELECT token FROM tokens WHERE clientid = %s"
-		self.cursor.execute(stmt, (str(userid),))
-		session_token = self.cursor.fetchall()
-		self.theDB.commit()
+		await cur.execute(stmt, (str(userid),))
+		session_token = await cur.fetchall()
+		await self.sqlBroker.close(cur)
```
Comment (xlatb) [inline modules/nsotoken.py]: Why "close" rather than "commit" here?

File: modules/nsotoken.py [hunk 2]
```diff
@@ -1,76 +1,74 @@
...
-	def get_session_token_mysql(self, userid):
+	async def get_session_token_mysql(self, userid):
+		cur = await self.sqlBroker.connect()
 		stmt = "SELECT session_token FROM tokens WHERE clientid = %s"
-		self.cursor.execute(stmt, (str(userid),))
-		session_token = self.cursor.fetchall()
-		self.theDB.commit()
+		await cur.execute(stmt, (str(userid),))
+		session_token = await cur.fetchall()
+		await self.sqlBroker.close(cur)
```
Comment (xlatb) [inline modules/nsotoken.py]: Why "close" rather than "commit" here?

Comment (Jetsurf) [reply modules/nsotoken.py]: Fixed :)

File: modules/nsotoken.py (hunk 1 above)
Comment (Jetsurf) [reply modules/nsotoken.py]: Fixed :)

**Pull request merged**